elastalert/config.py

```diff
@@ -158,6 +160,11 @@ def load_options(rule, conf, args=None):
     elif rule['timestamp_type'] == 'unix_ms':
         rule['ts_to_dt'] = unixms_to_dt
         rule['dt_to_ts'] = dt_to_unixms
+    elif rule['timestamp_type'] == 'custom':
+        def _dt_to_ts_with_format(dt):
+            return dt_to_ts_with_format(dt, ts_format=rule['timestamp_format'])
+        rule['ts_to_dt'] = ts_to_dt
```

Comment:
Do you need to create a custom ts_to_dt function? Otherwise, it's just using ``dateutil.parser.parse(timestamp)``, which may work for some formats, but not necessarily others. It could probably be ``datetime.datetime.strptime(ts, format)`` and then have the UTC timezone added by default like the normal ts_to_dt function does.